apps/src/netsim/NetSimRouterNode.js

```diff
@@ -79,6 +79,15 @@ var AUTO_DNS_HOSTNAME = 'dns';
 var AUTO_DNS_NOT_FOUND = 'NOT_FOUND';
 
 /**
+ * Maximum packet lifetime in the router queue, sort of a primitive Time-To-Live
+ * system that helps prevent a queue from being indefinitely blocked by a very
+ * large packet.  Packets that exceed this time will silently fail delivery.
+ * @type {number}
+ * @readonly
+ */
+var PACKET_MAX_LIFETIME_MS = 600000;
```

Comment:
somtimes i like to say 600 * 1000 so you can see how many seconds it is a bit more easily, esp when there are this many 0s.